spec/active_record_cloning_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'spec_helper'
+
+connect_to_sqlite
```

Comment:
Should this be in a before block? Even a before :all or before :suite block? Otherwise the simple act of requiring the file will connect to the database.